src/Node/index.tsx

```diff
@@ -119,22 +119,17 @@ export default class Node extends React.Component<NodeProps, NodeState> {
   // TODO: needs tests
   renderNodeElement = () => {
     const { data, hierarchyPointNode, renderCustomNodeElement } = this.props;
-    if (typeof renderCustomNodeElement === 'function') {
-      return renderCustomNodeElement({
+    const callable = renderCustomNodeElement === 'function' ? renderCustomNodeElement : DefaultNodeElement;
```

Comment:
Thanks for consolidating the prop specification 👍 

Suggested clearer naming + the existing `typeof` is missing which means this will always be `false`:

```suggestion
    const renderNode = typeof renderCustomNodeElement === 'function' ? renderCustomNodeElement : DefaultNodeElement;
```

Reply:
✅